File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
  Map<String, Map<String,Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterial;

  // public UberVan(String license, Driver driver, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
  //   super(license, driver);
  //   this.typeCarAccepted = typeCarAccepted;
  //   this.seatsMaterial = seatsMaterial;
  // }

  public UberVan(String license, Driver driver){
    super(license, driver);
  }

  @Override
  public Integer getPassengers() {
    return super.getPassengers();
  }

  public void setPassengers(Integer passengers) {
    if(passengers == 6){
      this.passengers = passengers;
    } else {
      System.out.println("Necesitas 6 pasajeros");
    }
  }

}

File: PHP/UberVan.php
<?php
class UberVan extends Car {
  public $typeCarAccepted;
  public $seatsMaterial;

  // public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial){
  //   parent:: __construct($license,$driver);
  //   $this -> typeCarAccepted = $typeCarAccepted;
  //   $this -> seatsMaterial = $seatsMaterial;
  // }
  public function __construct($license, $driver){
    parent:: __construct($license,$driver);
    
  }

  public function setPassengers($passengers) {
    
    if ($passengers == 6) {
        $this->passengers = $passengers;
    }
    else {
        echo "Necesitas asignar 6 pasajeros ";
    }

}
}
?>